assym/genassym.h

```diff
@@ -31,31 +31,33 @@
  */
 
 #ifndef _SYS_ASSYM_H_
-#define	_SYS_ASSYM_H_
+#define _SYS_ASSYM_H_
 
-#define	ASSYM_BIAS		0x10000	/* avoid zero-length arrays */
-#define	ASSYM_ABS(value)	((value) < 0 ? -((value) + 1) + 1ULL : (value))
+#define ASSYM_BIAS 0x10000 /* avoid zero-length arrays */
```

Comment:
We have changed method of generating `assym.h` in _mimiker_. You may consider a switch to [genassym.h](https://mimiker.ii.uni.wroc.pl/source/xref/mimiker/sys/script/genassym.sh).